frontend/app/routes/_public+/apply+/$id+/demographics-part1.tsx

```diff
@@ -26,58 +32,165 @@ export const meta: MetaFunction<typeof loader> = mergeMeta((args) => {
 });
 
 export async function loader({ request, params }: LoaderFunctionArgs) {
+  const { FIRST_NATIONS_YES_TYPE_ID } = getEnv();
   const bornTypes = await getLookupService().getAllBornTypes();
   const disabilityTypes = await getLookupService().getAllDisabilityTypes();
+  const firstNationsTypes = await getLookupService().getAllFirstNationsTypes();
+  const firstNationsEnumTypes = await getLookupService().getAllFirstNationsEnumTypes();
   const applyFlow = getApplyFlow();
   const { id, state } = await applyFlow.loadState({ request, params });
-  return json({ id, state, bornTypes, disabilityTypes });
+
+  const firstNationsYesCode = firstNationsTypes.find((firstNationsType) => firstNationsType.id === FIRST_NATIONS_YES_TYPE_ID);
+  if (!firstNationsYesCode) {
+    throw new Response(`Unexpected 'Other' gender type: ${FIRST_NATIONS_YES_TYPE_ID}`, { status: 500 });
```

Comment:
The message in the `Response` is misleading as it references `'Other'` gender type but we are checking for the existence of `firstNationsYesCode`.